### PR TITLE
Add workflow for testing both events and legacy

### PR DESCRIPTION
**Pull request by agouin**
Test both processors in github workflows

### CODE REVIEW

File: ibctest/ibc_test.go [hunk 1]
```diff
@@ -57,6 +58,23 @@ func TestRelayerDocker(t *testing.T) {
 	ibctestConformance(t, rf)
 }
 
+// TestRelayerDocker runs the ibctest conformance tests against
+// the current state of this relayer implementation built in docker.
+// Relayer runs using the event processor.
```
Comment (mark-rushakoff) [inline ibctest/ibc_test.go]: s/event/legacy/

File: ibctest/ibc_test.go [hunk 2]
```diff
@@ -57,6 +58,23 @@ func TestRelayerDocker(t *testing.T) {
...
+func TestRelayerDockerLegacyProcessor(t *testing.T) {
+	relayeribctest.BuildRelayerImage(t)
```
Comment (mark-rushakoff) [inline ibctest/ibc_test.go:67]: It seems like both of these should have `t.Parallel()` for when someone runs a broad `go test .` in the `ibctest` module. That would have no change against using the make targets from CI.

**Review by mark-rushakoff — APPROVED** [review]

File: ibctest/ibc_test.go (hunk 1 above)
Comment (agouin) [reply ibctest/ibc_test.go]: done

File: ibctest/ibc_test.go (hunk 2 above)
Comment (agouin) [reply ibctest/ibc_test.go:67]: done

**Pull request merged**